packages/components/src/Spinner/Spinner.css

```diff
@@ -0,0 +1,44 @@
+.spinner {
+  width: var(--space-larger);
+  height: var(--space-larger);
+  box-shadow: var(--shadow-dark);
+  border: solid var(--color-blue--light);
+  border-top-color: var(--color-white);
+  border-width: calc(var(--space-small) * 0.75);
+  border-radius: 50%;
+  vertical-align: middle;
+  background-color: var(--color-blue--lightest);
+  animation-duration: 1s;
+  animation-iteration-count: infinite;
+  animation-name: animation-spin;
+  animation-timing-function: linear;
+  will-change: transform;
+}
+
+@keyframes animation-spin {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(359deg);
+  }
+}
+
+/* -------------------------------------------------------------------------
+   Small Spinner
+   ------------------------------------------------------------------------- */
```

Comment:
I think that this component is small enough and the css is specific enough that we don't need the comments explaining them.